boards/px4/fmu-v6x/src/init.c

```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
  *
- *   Copyright (c) 2012-2020 PX4 Development Team. All rights reserved.
+ *   Copyright (c) 2012-2020, 2022 PX4 Development Team. All rights reserved.
```

Comment:
```suggestion
 *   Copyright (c) 2012-2022 PX4 Development Team. All rights reserved.
```

I would just do the full range, it's not like this file was never touched in 2021.